cli/tests/init_router.sh

```
@@ -5,22 +5,22 @@ fi
 mkdir -p output/init-router
 cd output/init-router
 
-RUST_BACKTRACE=1 cargo run --release init application router-test-node-application -p . -d postgresql -f prettier -l eslint -v zod -F express -r node -t vitest -m billing-base -m iam-base -D "Test service" -A "Rohin Bhargava" -L 'AGPL-3.0'
-RUST_BACKTRACE=1 cargo run --release init router rtr-test -p router-test-node-application/billing
+RUST_BACKTRACE=1 cargo run --release init application router-test-node-application -p ./router-test-node-application -o source -d postgresql -f prettier -l eslint -v zod -F express -r node -t vitest -m billing-base -m iam-base -D "Test service" -A "Rohin Bhargava" -L 'AGPL-3.0'
+RUST_BACKTRACE=1 cargo run --release init router rtr-test -p router-test-node-application/src/modules/billing
 
-cd router-test-node-application/billing
+cd router-test-node-application/src/modules/billing
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)